1636218959376	Sync.LogManager	DEBUG	Flushing file log
1636218959390	Sync.LogManager	DEBUG	Log cleanup threshold time: 1635354959390
1636218959395	Sync.LogManager	DEBUG	Done deleting files.
1636218961321	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1636219799500	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1636219799500	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1636219799504	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1636219799506	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1636225333675	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1636225333675	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1636225333677	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1636225333678	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1636225333678	Sync.Service	DEBUG	User-Agent: Firefox/93.0 (Linux x86_64) FxSync/1.95.0.20211004184504.desktop
1636225333678	Sync.Service	INFO	Starting sync at 2021-11-06 15:02:13 in browser session HSZmKI7QqiC8
1636225333679	Sync.Service	DEBUG	In sync: should login.
1636225333679	Sync.Service	INFO	User logged in successfully - verifying login.
1636225333679	Sync.SyncAuthManager	DEBUG	unlockAndVerifyAuthState has no FxA user
1636225333679	Sync.Status	DEBUG	Status.login: error.login.reason.no_username => error.login.reason.no_username
1636225333679	Sync.Status	DEBUG	Status.service: service.client_not_configured => service.client_not_configured
1636225333679	Sync.Service	DEBUG	Fetching unlocked auth state returned error.login.reason.no_username
1636225333679	Sync.ErrorHandler	ERROR	Sync encountered a login error
1636225333679	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1636225333679	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1636225333680	Sync.Service	DEBUG	Exception calling WrappedLock: Error: Login failed: error.login.reason.no_username(resource://services-sync/service.js:1038:15) JS Stack trace: onNotify@service.js:1038:15
1636225333680	Sync.Service	DEBUG	Not syncing: login returned false.
